src/main/java/com/mongodb/hibernate/internal/NotYetImplementedException.java

```diff
@@ -22,8 +22,8 @@
  * A temporary marker exception to denote that the feature in question is in the scope of MongoDB dialect but has not
  * been implemented yet.
  *
- * <p>Ultimately all of its references should be eliminated sooner or later, and then this class is supposed to be
- * deleted prior to product release.
+ * <p>TODO-HIBERNATE-45 Ultimately all of its references should be eliminated sooner or later, and then this class is
```

Comment:
The ticket: https://jira.mongodb.org/browse/HIBERNATE-45

Reply:
tbh, the ticket is not quite necessary for whenever we finished an undone feature, defintively we will remove the corresponding exception. If any feature is not finished, I am afraid we still need the exception linger there.

Reply:
This task will server as a reminder that we have unfinished business, and may eventually push us either towards implementing everything where this exception is thrown, or deciding that some of those places are not to be implemented at all, or coming up with a different more permanent solution.